### DISCUSSION TITLE
Permalink shortcut does not work

### DISCUSSION THREAD

**Discussion by lonix1** [community/community#140872] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

For the last few weeks I can't get a permalink by pressing <kbd>y</kbd>.

Don't know when this problem started. I upgraded ubuntu recently, but doubt that's the issue: other shortcuts work. There does not seem to be any conflicting browser extensions (I do use refined github, but same problem even when it's disabled), and I have no system shortcuts mapped to `y`.

Does anyone else have this problem?

environment: ubuntu 24.04, firefox and edge

**Comment by VeerammalV** [comment, marked as answer]
Ensure that GitHub is fully loaded before pressing 'y', as sometimes the shortcut won't work if the page isn't fully rendered.
Clear your browser cache or try using GitHub in incognito mode to rule out any caching issues.
If the problem persists across multiple browsers (like Firefox and Edge), it could be related to a GitHub or browser update. Try using a different browser or contacting GitHub support for further assistance.

**Reply by lonix1** [reply]
I found the issue.

The permalink shortcut works for a code file. But not for a repo's README.md; one must first load that file from the file list. Then the permalink shortcut works.

I already knew that, but forgot! :-)
